providers/base/units/power-management/jobs.pxu

```diff
@@ -134,6 +134,48 @@ _description:
  VERIFICATION:
      Did the system resume when the lid was opened?
 
+plugin: user-interact-verify
+category_id: com.canonical.plainbox::power-management
+id: power-management/lid_close_suspend_open
+estimated_duration: 20.0
+requires: device.product == 'Lid Switch'
+_description:
+ PURPOSE:
+    This test will check your lid sensor can detect lid close/open, and DUT will suspend when the lid closed
+ STEPS:
+    1. Press "Enter" to start the test
+    2. Close the lid (Plese close the lid within 10 sec)
+    3. Wait 5 seconds with the lid closed.
+    5. Open the lid
+ VERIFICATION:
+    Did the system suspend when lid closed, and resume back when the lid opened?
+command:
```

Comment:
The Tox output is not very clear, but if you go through, you will find the following FAIL logs:

```

In - line 4:
for i in {1..20}; do
^-^ SC2034: i appears unused. Verify use (or export if used externally).


In - line 9:
        for j in {1..20}; do
        ^-^ SC2034: j appears unused. Verify use (or export if used externally).


In - line 16:
                [ $now_suspend_times -gt $prev_suspend_times ] && exit 0 || sleep 0.5
                  ^----------------^ SC2086: Double quote to prevent globbing and word splitting.
                                         ^-----------------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
                [ "$now_suspend_times" -gt "$prev_suspend_times" ] && exit 0 || sleep 0.5

For more information:
  https://www.shellcheck.net/wiki/SC2034 -- i appears unused. Verify use (or ...
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
test_job_command_units/power-management/jobs.pxu:137-177_power-management/lid_close_suspend_open (plainbox.provider_manager.InlineShellcheckTests) ... FAIL
```

FYI, the Tox script will run different tests on Checkbox and its providers:

- unit tests
- providers validation (to make sure the jobs are valid, no missing or duplicate fields, etc.)
- shell checks for scripts and jobs' `command` fields

Fix the script in your command with the recommendations from shellcheck, and resubmit.